panther_lights/src/apa102.cpp

```diff
@@ -0,0 +1,102 @@
+#include <panther_lights/apa102.hpp>
+
+#include <fcntl.h>
+#include <unistd.h>
+#include <linux/spi/spidev.h>
+#include <sys/ioctl.h>
+
+#include <cmath>
+#include <cstdint>
+#include <fstream>
+#include <stdexcept>
+#include <string>
+#include <vector>
```

Comment:
```suggestion
#include <cmath>
#include <fstream>
#include <stdexcept>
```
This are already included in hpp file. Maybe we should consider moving all includes to header file?

Reply:
You already proved yourself that they should be in both places